pilot/pkg/config/kube/gateway/testdata/mesh.status.yaml.golden

```diff
@@ -171,9 +171,9 @@ status:
   parents:
   - conditions:
     - lastTransitionTime: fake
-      message: Route was valid
-      reason: Accepted
-      status: "True"
+      message: 'parent service: "echo-port" not found'
```

Comment:
I think we are accidentally turning these test cases that were supposed to be valid into invalid cases so we are basically losing what we wanted to test. We should either change the names to existing service names, or add them as "known services" in the test setup: https://github.com/istio/istio/blob/03f417c38b0066b5e85903a26603a67b797819bb/pilot/pkg/config/kube/gateway/conversion_test.go#L71

Reply:
I will add the missing services.